src/pages/VolunteerDashboardPage.jsx

```diff
@@ -71,6 +72,55 @@ const VolunteerDashboardPage = () => {
     }
   }, []);
 
+  useEffect(async () => {
+    try {
+      const [notificationsRes] = await Promise.all([
+        OCHBackend.get('/notification/', { withCredentials: true }),
+      ]);
+      setUserNotifications(notificationsRes.data);
+    } catch (err) {
+      // TODO: handle error
+      // eslint-disable-next-line no-console
+      console.log(err);
+    }
+  }, []);
+
+  const Notifications = () => {
+    return userNotifications.map(notification => (
+      <Toast
+        key={notification._id}
+        title={notification.title}
+        description={notification.message}
+        status={notification.type === 'MONITOR_LOG_APPROVED' ? 'success' : 'error'}
+        variant="left-accent"
+        closeButton={notification.type === 'MONITOR_LOG_APPROVED' ? 'true' : 'false'}
```

Comment:
When a notification is closed, it should delete that notification

Reply:
Ok wait I just realized,

These notifications are going to have to be created when an admin approves a log, that way it's stored in the backend end and so that we can delete it once the user closes it. We don't have approval functionality yet so you're good.